diff --git a/target/linux/mediatek/filogic/base-files/lib/upgrade/platform.sh b/target/linux/mediatek/filogic/base-files/lib/upgrade/platform.sh
index a60350b67d..328ba4728d 100755
--- a/target/linux/mediatek/filogic/base-files/lib/upgrade/platform.sh
+++ b/target/linux/mediatek/filogic/base-files/lib/upgrade/platform.sh
@@ -66,6 +66,9 @@ platform_do_upgrade() {
        local board=$(board_name)
 
        case "$board" in
+       cmcc,rax3000m|\
+       cmcc,rax3000me|\
+       jcg,q30-pro|\
         qihoo,360t7)
                CI_UBIPART="ubi"
                CI_KERNPART="kernel"
@@ -79,13 +82,10 @@ platform_do_upgrade() {
        bananapi,bpi-r4-poe|\
        cetron,ct3003-ubootmod|\
        cmcc,a10-ubootmod|\
-       cmcc,rax3000m|\
-       cmcc,rax3000me|\
        cudy,tr3000-v1-ubootmod|\
        gatonetworks,gdsp|\
        h3c,magic-nx30-pro|\
        imou,lc-hx3001|\
-       jcg,q30-pro|\
        konka,komi-a31|\
        livinet,zr-3020-ubootmod|\
        mediatek,7981r128|\